posthog/hogql/database/database.py

```diff
@@ -261,10 +262,15 @@ def create_hogql_database(
         DataWarehouseTable,
     )
 
-    if timings is None:
-        timings = HogQLTimings()
+    with timings.measure("team"):
```

Comment:
If both are provided, want to throw if team.id is different to team_id?

Reply:
Yeah, can add that.

Reply:
Done, and fixed the `timings` bug as well